datastore/badger/v4/datastore.go

```diff
@@ -622,18 +621,17 @@ func (t *txn) query(q dsq.Query) (dsq.Results, error) {
 	}
 
 	it := t.txn.NewIterator(opt)
-	qrb := dsq.NewResultBuilder(q)
```

Comment:
question: Is this urgent?  This is just going to create a whole bunch of conflicts for me and will be removed anyway

Reply:
Reverted 👍

Reply:
Thanks Fred